projects/serenity/project.yaml

```diff
@@ -7,6 +7,8 @@ auto_ccs:
   - "ali.mpfard@gmail.com"
   - "luke.wilde@live.co.uk"
   - "bugaevc@serenityos.org"
+  - "b.gianfo@gmail.com"
+  - "~awesomekling/serenityos-dev@lists.sr.ht"
```

Comment:
Just confirming: Is "~awesomekling/serenityos-dev@lists.sr.ht" an email address? it looks very weird to me.

Reply:
Yes, that's the format of mailing lists hosted on: http://sr.ht 
See: https://lists.sr.ht/~awesomekling/serenityos-dev